quartz/function_job_test.go

```diff
@@ -13,14 +14,14 @@ func TestFunctionJob(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	var n = 2
+	var n int32 = 2
 	funcJob1 := quartz.NewFunctionJob(func(_ context.Context) (string, error) {
-		n += 2
+		atomic.AddInt32(&n, 2)
```

Comment:
I'd be inclined to use the `atomic.Int64` type/wrapper which ends up being clearer/easier...

Reply:
Agree. But it is available starting from go1.19.